CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ Changelog
 =========
 
 - added `reachedBottom(offset:)` for `UIScrollView`
+- removed deprecated `OSAtomicOr32OrigBarrier`
```

Comment:
`once` now uses a `NSRecursiveLock` instead of the deprecated `OSAtomicOr32OrigBarrier`